packages/api/endpoints/instance-meta.js

```diff
@@ -14,6 +14,7 @@ function instanceMetadata(req, res) {
     cmr: {
       provider: process.env.cmr_provider,
       environment: process.env.CMR_ENVIRONMENT || 'UAT',
+      oauth_provider: process.env.cmr_oauth_provider || '',
```

Comment:
maybe I should make "Earthdata" the OR because that is technically the default

Reply:
No, I just slacked you somthing like this:  I’m not sure it should switch.  This metadata endpoint probably shouldn’t even return ‘UAT’ if process.env.CMR_ENVIRONMENT is not set.
 This endpoint should represent how cumulus is actually configured, and the defaults should flow to here, if not, we don't want to lie to the user.